iree/compiler/Dialect/LinalgExt/IR/TiledOpInterface.td

```diff
@@ -54,6 +54,24 @@ def TiledOpInterface : OpInterface<"TiledOpInterface"> {
         /*defaultImplementation=*/[{
           return nullptr;
         }]
+      >,
+      InterfaceMethod<
+        /*desc=*/[{
+          Generates the loop body implementation. Assume that all the parallel
+          loops and reduction loops are created and the insertion point of the
+          build is set to the innermost of the loop. This method implements the
+          loop body IRs.
+        }],
+        /*retType=*/"void",
```

Comment:
To be future safe, just return `LogicalResult` with default implementation being `return failure();` . If an op doesnt have an implementation to lower to loops then trying to do that should be a failure.

Also maybe `generateScalarImplementation` is a slightly better name.